.github/workflows/tests.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches:
       - master
-    types: [opened, synchronize]
+    types: [ opened, synchronize ]
```

Comment:
whitespace change. does it fix something? or just a refactor?

Reply:
just gofmt work, It wasn't my plan, reverted